modelscope/models/cv/dense_optical_flow_estimation/raft_model.py

```diff
@@ -0,0 +1,51 @@
+import os.path as osp
+
+import torch
+
+from modelscope.metainfo import Models
+from modelscope.models.base.base_torch_model import TorchModel
+from modelscope.models.builder import MODELS
+from modelscope.models.cv.dense_optical_flow_estimation.core.raft import RAFT
+from modelscope.outputs import OutputKeys
+from modelscope.utils.constant import ModelFile, Tasks
+import argparse
+
+@MODELS.register_module(
+    Tasks.dense_optical_flow_estimation,
+    module_name=Models.raft_dense_optical_flow_estimation)
+class DenseOpticalFlowEstimation(TorchModel):
+    def __init__(self, model_dir: str, **kwargs):
```

Comment:
this implementation should be the implementation of dense_optical_flow_estimation_raft pipeline